package/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+applications:
+  - openshift4-service-mesh
+  - openshift4-operators as openshift-distributed-tracing
+  - openshift4-operators as openshift-kiali
+
+parameters:
+  components:
+    openshift4-operators:
+      url: https://github.com/appuio/component-openshift4-operators
+      version: feat/manage-subscription
+    openshift4-service-mesh:
+      url: https://github.com/appuio/component-openshift4-service-mesh
+      version: initial-implementation
```

Comment:
This is not what we actually want, but necessary for the tests to pass, since there's currently no way to indicate that this component lives in the same repo as the package.

Reply:
Removed from this PR for now, will introduce the package in a follow-up to reduce the amount of cleanup required once we create a tagged release.